Project.toml

```diff
@@ -8,6 +8,7 @@ version = "0.12.10"
 Base64 = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 Distributed = "8ba89e20-285c-5b6f-9357-94700520ee1b"
+FuzzyCompletions = "fb4132e2-a121-4a70-b8a1-d5b831dcdcc2"
```

Comment:
Can you also add a `[deps]` entry?

Reply:
you mean, `[compat]` entry ?
`FuzzyCompletions` is already in `[deps]` section:
```toml
```

Reply:
ah right yes

Reply:
done